dcpy/library/templates/nysdec_freshwater_wetlands_checkzones.yml

```diff
@@ -0,0 +1,25 @@
+dataset:
+  name: nysdec_freshwater_wetlands
+  acl: public-read
+  source:
+    arcgis_feature_server:
+      server: nys_clearinghouse
+      name: State_Regulated_Freshwater_Wetlands
+      layer: 1
+    geometry:
+      SRS: EPSG:2263
```

Comment:
[This](https://services6.arcgis.com/DZHaqZm9cxOD4CWM/ArcGIS/rest/services/State_Regulated_Freshwater_Wetlands/FeatureServer/1) says `Spatial Reference: 26918 (26918)`. Does that matter?

Reply:
I believe that's meant for the extent. Currently, this geometry gets fed into the call to arcgis connector, and queries for the data specifically in the given SRS

Reply:
The REST service supports that. So to answer your question - not entirely, though it's maybe not great that there's this uncertainty about where transformation happens